Example/Tests/DiscoveryTaskTests.swift

```diff
@@ -0,0 +1,97 @@
+import XCTest
+@testable import OktaAuth
+
+class DiscoveryTaskTests: XCTestCase {
+
+    var apiMock: OktaApiMock!
+    
+    override func setUp() {
+        super.setUp()
+        apiMock = OktaApiMock()
+    }
+
+    override func tearDown() {
+        apiMock = nil
+        super.tearDown()
+    }
+    
+    func testRunSucceeded() {
+        apiMock.configure(response: self.validOIDConfigDictionary)
+        
+        runAndWaitDiscovery(config: validConfig) { oidConfig, error in
+            XCTAssertNil(error)
+            XCTAssertNotNil(oidConfig)
```

Comment:
nit: Asserting whether something is defined or not isn't usually enough. Instead, lets validate that the `oidConfig` object matches our expected object using `XCTAssertEquals`

Reply:
Done.